modules/core/src/passes/screen-pass.js

```diff
@@ -17,6 +17,8 @@ export default class ScreenPass extends Pass {
   render(params) {
     const gl = this.gl;
 
+    setParameters(gl, {viewport: [0, 0, gl.drawingBufferWidth, gl.drawingBufferHeight]});
+
```

Comment:
Why do you need to use `setParameters` rather that adding to the state changes within `withParameters`?

Reply:
The idea is that `viewport` must be set before each pass any ways. Using `withParameter` it is reset after render, which result in two GPU calls instead of one. It is more important in the layers pass as an optimization, but not that important here.